src/main/java/org/traccar/protocol/PretraceProtocolDecoder.java

```diff
@@ -75,8 +77,8 @@ protected Object decode(
         Position position = new Position(getProtocolName());
         position.setDeviceId(deviceSession.getDeviceId());
 
+        position.set(Position.KEY_GPS, parser.nextInt(0));
```

Comment:
What is this?

Reply:
This is a bitvalue indicating the GPS type:
![image](https://user-images.githubusercontent.com/51079911/111219799-b3befb00-85d8-11eb-9023-f7fada1ece75.png)